pyne/mcnp.py

```diff
@@ -1231,7 +1231,7 @@ def read_variable_ids(self):
 
         if self.eightbytes:
             mcnp_version = self.mcnp_version_info[8:13]
```

Comment:
Is it more robust to `strip()` the whitespace from the end of this?
```suggestion
            mcnp_version = self.mcnp_version_info[8:13].strip()
```

Reply:
Sure, but I like @zxkjack123 to make the change and what he thinks.